1738277151158	Sync.LogManager	DEBUG	Flushing file log
1738277151159	Sync.LogManager	DEBUG	Log cleanup threshold time: 1737413151159
1738277151163	Sync.LogManager	DEBUG	Done deleting files.
1738292533864	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1738292534593	Sync.SyncScheduler	DEBUG	Woke from sleep.
1738292534754	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1738292534764	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1738292535194	Sync.Service	DEBUG	User-Agent: Firefox/134.0.2 (Windows NT 10.0; Win64; x64) FxSync/1.136.0.20250120135430.desktop
1738292535194	Sync.Service	INFO	Starting sync at 2025-01-31 10:02:15 in browser session 6DKUZYjb8LNM
1738292535194	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1738292535196	Sync.Status	INFO	Resetting Status.
1738292535196	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1738292535197	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1738292535197	FirefoxAccounts	DEBUG	getOAuthToken enter
1738292535197	FirefoxAccounts	TRACE	getCachedToken returning cached token
1738292535197	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1738292535197	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1738292535197	Sync.SyncAuthManager	INFO	Getting sync key
1738292535197	FirefoxAccounts	DEBUG	getOAuthToken enter
1738292535197	FirefoxAccounts	TRACE	getCachedToken returning cached token
1738292535197	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1738292535198	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1738292535198	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1738292535198	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1738292535198	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1738292535257	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1738292535257	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1738292535287	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1738292535287	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1738292535287	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1738292535287	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1738292535287	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/open-uri
1738292535287	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1738292535288	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1738292535288	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/close-uri/v1
1738292535344	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/266395372/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1738292535344	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1738292535344	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1738292535344	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1738292535345	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1738292535345	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1738292535349	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
